_rules/element-lang-valid-de46e4.md

```diff
@@ -25,22 +25,21 @@ acknowledgments:
 
 ## Applicability
 
-This rules applies to any HTML element that:
+This rules applies to any HTML element with a `lang` [attribute value][] that is not empty (`""`) and for which all of the following is true:
 
-- has a [node document](https://dom.spec.whatwg.org/#concept-node-document) with a [content type](https://dom.spec.whatwg.org/#concept-document-content-type) of `text/html`; and
-- is a [descendant](https://dom.spec.whatwg.org/#concept-tree-descendant) in the [flat tree](https://drafts.csswg.org/css-scoping/#flat-tree) of a `body` element; and
-- has a `lang` attribute that is not empty ("").
+- the element is a [descendant][] in the [flat tree][] of a `body` element; and
+- the element has a [node document][] with a [content type][] of `text/html`.
+- the element has a [visible][] [text node][] as a [descendant][] in the [flat tree][].
```

Comment:
I think this should be in line with what I did here:

https://github.com/act-rules/act-rules.github.io/pull/1334/files#diff-53f178e37d6189f50875615ad27cecddR20

```suggestion
- the element has a [text node][] as a [descendant][] in the [flat tree][] that is [visible][] or [included in the accessibility tree][].
```